ml/train_model.py

```diff
@@ -94,6 +85,22 @@
 if yaml_dict is None:
     raise RuntimeError("File config.yaml not found.")
 
+# Connect to the MongoDB database with read-write access
+db_host = yaml_dict["database"]["host"]
+db_name = yaml_dict["database"]["name"]
+db_auth = yaml_dict["database"]["auth"]
+db_username = yaml_dict["database"]["username_rw"]
+db_password_env = yaml_dict["database"]["password_rw_env"]
+db_password = os.getenv(db_password_env)
+if db_password is None:
+    raise RuntimeError(f"Environment variable {db_password_env} must be set!")
```

Comment:
I wonder if this is equivalent to what we had before, which in the new code would read
```py
db_password = re.findall(f"{db_password_env}='(.+)'", db_profile)[0]
```

If it is equivalent, we can remove `db_profile` from the code. If it is not equivalent, I can change the code following the suggestion here.

Reply:
Resolved in b0e16b8.